packages/jsapi-types/src/dh.types.ts

```diff
@@ -558,6 +558,7 @@ export interface Column {
   readonly constituentType: string;
 
   readonly isPartitionColumn: boolean;
+  readonly isSortable: boolean;
```

Comment:
Need this to be optional because we do not have it implemented on Enterprise.
```suggestion
  readonly isSortable?: boolean;
```